examples/taskruns/task-volume.yaml

```diff
@@ -31,3 +27,7 @@ metadata:
 spec:
   taskRef:
     name:  task-volume
+  podTemplate:
+    volumes:
+    - name: custom
+      emptyDir: {}
```

Comment:
hee hee nice :D

Reply:
actually this fails the validation on Task as `volumes` is no more declared there. I am wondering if I should revert this part (aka adding the volume), and do that in a follow-up PR (just to not overload this PR with too much changes)

Reply:
Fixed :angel: